relay/kusama/src/lib.rs

```diff
@@ -684,6 +683,8 @@ impl pallet_staking::Config for Runtime {
 	type NominationsQuota = pallet_staking::FixedNominationsQuota<{ MaxNominations::get() }>;
 	type MaxUnlockingChunks = frame_support::traits::ConstU32<32>;
 	type HistoryDepth = frame_support::traits::ConstU32<84>;
+	// TODO:(PR#159) check/set correct constant and remove this comment!
+	type MaxControllersInDeprecationBatch = ConstU32<751>;
```

Comment:
Hey @rossbulat, as an author of https://github.com/paritytech/polkadot-sdk/pull/2589, could you please confirm or adjust correct value of `MaxControllersInDeprecationBatch` for Kusama and Polkadot?

Reply:
No problem @bkontur, I've left comments with the up to date figures. Thank you for flagging.